packages/thumbprint-react/components/Dropdown/index.tsx

```diff
@@ -27,7 +27,7 @@ const getUIState = ({
     return 'default';
 };
 
-interface PropTypes {
```

Comment:
This used to be just `string` but now it's `string | number`. Is number definitely a valid value type?

Reply:
Yeah, input value can actually be of type `value?: string | ReadonlyArray<string> | number | undefined;`, but I figured we don't want to support the `string[]` case for now